seat.c

```diff
@@ -260,7 +261,8 @@ handle_key_event(struct wlr_input_device *device, struct cg_seat *seat, void *da
 
 	bool handled = false;
 	uint32_t modifiers = wlr_keyboard_get_modifiers(device->keyboard);
-	if ((modifiers & WLR_MODIFIER_ALT) && event->state == WLR_KEY_PRESSED) {
+	if ((modifiers & WLR_MODIFIER_ALT) &&
+	    (enum wl_keyboard_key_state) event->state == WL_KEYBOARD_KEY_STATE_PRESSED) {
```

Comment:
Is that enum cast really needed?

```suggestion
	    event->state == WL_KEYBOARD_KEY_STATE_PRESSED) {
```

```c
/* a bit earlier on line 254 */
struct wlr_event_keyboard_key *event = data;
```

```c
/* from include/wlr/types/wlr_keyboard.h */
struct wlr_event_keyboard_key {
    uint32_t time_msec;
    uint32_t keycode;
    bool update_state; // if backend doesn't update modifiers on its own
    enum wl_keyboard_key_state state;
};
```